modules/nodes-secondary-scheduler-install-console.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-configuring.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-secondary-scheduler-install-console_{context}"]
+= Installing the {secondary-scheduler-operator}
+
+You can use the web console to install the {secondary-scheduler-operator-full}.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Create the required namespace for the {secondary-scheduler-operator-full}.
+.. Navigate to *Administration* -> *Namespaces* and click *Create Namespace*.
+.. Enter `openshift-secondary-scheduler-operator` in the *Name* field and click *Create*.
++
+// There are no metrics to collect for the secondary scheduler operator as of now, so no need to add the metrics label
++
+// TODO: should the UI prevent you from selecting a different ns if this one is required?
```

Comment:
@ingvagabund Same question here about the UI preventing you from selecting any other namespace (later in the procedure) when installing the operator if this one is hardcoded/required?

Reply:
The same answer applies.